app/views/support/PlanOps.scala

```diff
@@ -92,6 +86,25 @@ object PlanOps {
     }
```

Comment:
`productType` just above this line should also be in ProductOps, but Patricio is working on this code too and probably going to do that, so avoiding conflicts in this PR